### ISSUE TITLE
Can't access MkStorage in BundleTests

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#739]
It's not possible to prepare some data in `Github` instance via `MkStorage` for bundle tests, because it accessed with `ExtGithub(farm).value()` which creates `MkGithub` instance for tests with default storage.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by amihaiemil** [comment]
@g4s8 Yes, I also had this issue. In fact, I believe we should get rid of ExtGithub (why do we even need ``Scalar<Github>``) -- it doesn't do enything except taking a ``Farm`` in its ctor and passing its ``Props`` to RtGithub. And, well, it gives us trouble when we need MkGithub...

**Comment by g4s8** [comment]
@amihaiemil `ExtGithub` can construct proper `Github` instance with credentials from properties for production and `MkGithub` for test, I think we need similar class anyway, how would you remove it?

**Comment by amihaiemil** [comment]
@g4s8 First off, I would at least rename if (wtf is ``Ext``?). 

Then, I would add an implementation of ``Github``, which would work with an authenticated ``RtGithub`` behind, like this:

``Github github = new FarmGithub(farm)`` -- this encapsulates and delegates all operations to an ``RtGithub`` authenticated via the Props from the farm. Maybe it is enough to pass the Props directly, instead of the whole farm. 

and then, in tests I would instantiate MkGithub where needed.

I think the whole confusion comes from the fact that testing code/infrastructure is mixed with the actual, production code. I've seen in a few places checks for ``testing`` prop... this is not ok at all :)

**Comment by yegor256** [comment]
@amihaiemil yes, we can do `new FarmGithub()` in this particular case, but there are many other `Ext*` classes, which are basically singletons. That's why all of them are designed in the same manner, as singletons.
